### PR TITLE
added enw page layout for agent resource access

### PR DESCRIPTION
**Pull request by norbert989**
# Applied new layout for the Agent Access resource page

# Checklist

- [x] All acceptance criteria are met.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: components/profile/personAvatar/styles.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const styles = (theme) => {
+  return createStyles(theme, ["back-to-nav", "input"], {
+    avatar: {
+      width: "64px",
+      height: "64px",
+    },
+    avatarText: {
+      fontFamily: theme.typography.h1.fontFamily,
+      fontSize: "1.5rem",
+      color: "#000",
```
Comment (VirginiaBalseiro) [inline components/profile/personAvatar/styles.js:33]: Can we pull this from the theme?

**Review by VirginiaBalseiro — APPROVED** [review]
looks good!

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
I just noticed this doesn't work with apps, you can see how the profile link is built here: https://github.com/inrupt/pod-browser/pull/313

**Comment by VirginiaBalseiro** [comment]
Also, I just realized we don't have a specific subtask for the styling of the resource table so it should be done as part of this.

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
When I visit a person's profile I get directed to the `app` path and therefore the app profile is shown instead of the person's (the path should be `person`):

image

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
Also missing the message "Agent has access to these resources" above the table: 

image

And could we add a spinner while resources are loading in the table? It's a bit disconcerting when one first loads the page and it takes a bit to display the resources.

File: components/profile/appAvatar/styles.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      height: "64px",
+      background: "transparent",
+    },
+    appAvatar: {
+      textAlign: "center",
+      padding: theme.spacing(1),
+      borderRadius: "50%",
+      fontSize: theme.typography.h5.fontSize,
+      color: theme.palette.primary.main,
+      backgroundColor: "#F1EDFF",
```
Comment (VirginiaBalseiro) [inline components/profile/appAvatar/styles.js:37]: Can we put this color in a constant inside the same file? This way we can track later which colors we're reusing and should be added to the theme.

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]

File: components/profile/appAvatar/styles.js (hunk 2 above)
Comment (norbert989) [reply components/profile/appAvatar/styles.js]: This color was actually used a few times before so I added the const in all places and marked with a TODO

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by KyraAssaad — DISMISSED** [review]
It looks great! A few things: 
1. It looks like the table sorting on the Access column isn't working correctly
Inrupt_PodBrowser
I'd be happy to remove the sorting on the column if that's easier.

2. There's a weird thing that happens when you are scrolled to the bottom of the page and then click on a resource to open the sidebar- once the content in the sidebar has loaded, the top of the sidebar where the Close button is disappears. You can see it again if you scroll up a bit, but it's disorienting if you don't know to do that. I can't replicate the behaviour on the Files page- the top of the sidebar with the Close button is always visible no matter how far you scroll down the page (although it seems that when you open the sidebar, the page jumps back up to the top so that might be why). Let me know if you want to chat quickly so I can show you. If it's not an easy fix, we can create a ticket to address it later.

**Pull request merged**